src/core/init/initialize_media_source.ts

```diff
@@ -425,9 +425,13 @@ export default function InitializeOnMediaSource(
                 // to flush the buffers
                 const { position } = evt.value;
                 if (position + 0.001 < evt.value.duration) {
-                  mediaElement.currentTime += 0.001;
+                  clock$.pipe(take(1), tap((tick) => {
+                    tick.setCurrentTime(mediaElement.currentTime + 0.001);
```

Comment:
I don't think this will ever happen, as the returned obs by the pipe function is not subscribed.

Reply:
I resolve the thread since it changed, and Im no longer using that way.

Reply:
Done